pkg/kubelet/cpumanager/policy.go

```diff
@@ -24,6 +24,7 @@ import (
 // Policy interface used by cpu manager, shall implement logic for cpu to pod
 // assignment
 type Policy interface {
+	IsUnderPressure() bool
```

Comment:
The thing is that this has to be reachable from the kubelet level for `setNodeCPUPressureCondition()` and that only has access to the Manager.  So we have to plumb it through.

Also assumes that the state layer has sufficient information to determine CPU pressure or not.  Seems like that is a Policy level decision.